manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "store",
-  "version": "2.22.4",
+  "version": "2.21.4",
```

Comment:
I think you need to rebase this branch

Reply:
I think I've successfully rebased this branch now, I notice the file conflict warning in the pull request has gone away. This is my first time using github so please let me know if I've done something wrong or if you have any more suggestions for things I need to do to my branch.

Reply:
Sorry for the delay, but I'm afraid you need to rebase again.